tutorial/markdown/nodejs/ottoman-express.md

```diff
@@ -21,120 +21,223 @@ sdk_language:
+### Directly on Local Machine
 
-Now we are ready to run our application:
+At this point, we have installed the dependencies, loaded the travel-sample data and configured the application with the credentials. The application is now ready and you can run it.
 
-```shell
+The application will run on port 3000 of your local machine (http://localhost:3000). You will find the Swagger documentation of the API which you can use to try the API endpoints.
+
+```sh
+# Execute this command in the project's root directory
 npm start
 ```
+### Docker
+
+If you prefer to run this quick start using Docker, we have provided the Dockerfile which you can use to build the image and run the API as a container.
+
+- Build the Docker image
+
+```sh
+cd src
+docker build -t couchbase-ottoman-quickstart .
+```
+
+- Run the Docker image
+
+```sh
+docker run -it --env-file config/dev.env -p 3000:3000 couchbase-ottoman-quickstart
+```
+
+> Note: The `config/dev.env` file has the connection information to connect to your Capella cluster. With the `--env-file`, docker will inject those environment variables to the container.
+
+Once the app is up and running, you can launch your browser and go to the [Swagger documentation](https://localhost:3000/) to test the APIs.
+
+### Verifying the Application
+
+Once the application starts, you can see the details of the application on the logs.
+
+![Application Startup](ottoman_app_startup.png)
+
+The application will run on port 3000 of your local machine (http://localhost:3000). You will find the interactive Swagger documentation of the API if you go to the URL in your browser. Swagger documentation is used in this demo to showcase the different API end points and how they can be invoked. More details on the Swagger documentation can be found in the [appendix](#swagger-documentation).
+
+![Swagger Documentation](ottoman_swagger_documentation.png)
 
-If your database is set up correctly, we will now have an API running on port 3000 as specified in the `dev.env` file. Let's go over what we have done to create this API demo application.
+## Data Model
 
-## What We'll Cover
+For this tutorial, we use three collections, `airport`, `airline` and `route` that contain sample airports, airlines and airline routes respectively. The route collection connects the airports and airlines as seen in the figure below. We use these connections in the quickstart to generate airports that are directly connected and airlines connecting to a destination airport. Note that these are just examples to highlight how you can use SQL++ queries to join the collections.
+![img](travel_sample_data_model.png)
 
-A simple REST API using Express and [Ottoman JS](https://ottomanjs.com), an object-document mapping built on top of the [Couchbase NodeJS SDK v3.x](https://docs.couchbase.com/nodejs-sdk/current/hello-world/start-using-sdk.html) with the following endpoints:
+## Let Us Review the Code
```

Comment:
Can you add a newline before the image to not wrap the image around the text?

Reply:
Updated to add a newline before image